model/task/task.go

```diff
@@ -779,6 +779,8 @@ func displayTaskPriority(status string) int {
 		return 80
 	case evergreen.TaskSystemUnresponse:
 		return 90
+	case evergreen.TaskSetupFailed:
+		return 95
```

Comment:
I'd put this at 55, I think, maybe as high as 75, but 95 seems too high.

Reply:
The sort is such that the lowest priority dictates the status of the display task. So at 55, a display task with both a timed out test and a setup failure will display as a setup failure. Is that what you were thinking? I think this is in the correct place

Reply:
yeah, makes sense.